vivisect/analysis/elf/elfplt.py

```diff
@@ -22,16 +66,99 @@ def analyze(vw):
             continue
         analyzePLT(vw, sva, ssize)
 
+gots = {}
```

Comment:
So I'm *super* wary about keeping global state like this, especially if you've got multiple workspaces running like you do in the tests. I'd be very wary about this.

But it was this and the plts thing below that I changed to get the tests passing. And since there are no callers of getPLTs, and the fact that since I can comment out the tests until it's just the libstdc test running (and that works), but with all of them enabled it fails, I've got a strong suspicion that keeping around global state like this is causing a lot of problems.

Reply:
and you are *right* to be wary.